File: THOK/Fusion创建新模块所需的文件列表和文件/src/User Interface/Fusion/Extjs/Fusion/classic/src/Warehouse/model/Bill/ManualMaterialBills/ManualMaterialBillModel.js
/*
* 
*/
Ext.define('Fusion.warehouse.model.bill.ManualMaterialBills.ManualMaterialBillModel', {
    extend: 'Ext.data.Model',
    idProperty: 'ManualMaterialBillNo',

    requires: [
        'Ext.data.proxy.Rest'
    ],

    fields: [
        { name: 'ManualMaterialBillNo', type: 'string' },
        {
            name: 'PlanBeginTime', type: 'string',
            convert: function (value) {
                if (null != value) {
                    return Ext.Date.format(new Date(value), 'Y/m/d H:i:s');
                } else {
                    return null;
                }
            }
        },
        {
            name: 'PlanEndTime', type: 'string',
            convert: function (value) {
                if (null != value) {
                    return Ext.Date.format(new Date(value), 'Y/m/d H:i:s');
                } else {
                    return null;
                }
            }
        },
        { name: 'ManualMaterialBillStatus', type: 'string' },
        { name: 'Maker', type: 'string' },
        { name: 'MakeTime', type: 'string' },
        { name: 'Auditor', type: 'string' },
        { name: 'AuditTime', type: 'string' },
        { name: 'OriginPlanNo', type: 'string' },
        { name: 'ManualMaterialBillOrigin', type: 'string' },
        { name: 'CreateTime', type: 'string' },
        { name: 'UpdateTime', type: 'string' },
        { name: 'RowVersion', type: 'string' }
    ],

    proxy: {
        type: 'rest',

        api: {
            read: 'Api/ManualMaterialBill/Find',
            create: 'Api/ManualMaterialBill/Add',
            update: 'Api/ManualMaterialBill/Modify',
            destroy: 'Api/ManualMaterialBill/Remove'
        }
    },

    getName: function () {
        return this.get('ManualMaterialBillNo')
    }
});

File: THOK/Fusion创建新模块所需的文件列表和文件/src/Application/Fusion.Context.Warehouse.Application/Models/Bill/ManualMaterialBills/ManualMaterialBillModel.cs
using System;
using System.Collections.Generic;
using Fusion.Context.Warehouse.Domain.Models.Bill.ManualMaterialBills;

/*
* 2020-07-31 Create by wuzuqiang
*/
namespace Fusion.Context.Warehouse.Application.Models.Bill.ManualMaterialBills
{
    public class ManualMaterialBillModel
    {
        public string ManualMaterialBillNo { get; set; }
        public string OriginPlanNo { get; set; }
        public DateTime PlanBeginTime { get; set; }
        public DateTime PlanEndTime { get; set; }
        public string BillNo { get; set; }
        public string Maker { get; set; }
        public DateTime MakeTime { get; set; }
        public string Auditor { get; set; }
        public DateTime? AuditTime { get; set; }
        public ManualMaterialBillStatus ManualMaterialBillStatus { get; set; }
        public DateTime CreateTime { get; set; }
        public DateTime UpdateTime { get; set; }
        public byte[] RowVersion { get; set; }

        public string TeamCode { get; set; }
        public string TeamName { get; set; }
        public string MachineCode { get; set; }
        public string MachineName { get; set; }
        public string BrandName { get; set; }
        public DateTime? StartTime { get; set; }
        public DateTime? FinishTime { get; set; }

        public string ManualMaterialBillStatusName { get; set; }
        public IList<ManualMaterialBillDetailModel> ManualMaterialBillDetails { get; set; }
    }
}
